looprpc/client.proto

```diff
@@ -133,6 +133,14 @@ message LoopOutRequest {
     should be swept within.
     */
     int32 sweep_conf_target = 9;
+
+    /**
+    The maximum time (in seconds) we allow the server to wait before publishing
+    the HTLC on chain. Setting this to a larger value will give the server the
+    opportunity to batch multiple swaps together, and wait for low-fee periods
+    before publishing the HTLC, potentially resulting in a lower swap fee.
+    */
+    uint64 max_swap_wait_time_sec = 10;
```

Comment:
As mentioned before, convert to absolute (unix) timestamp to support absolute deadlines.